MultiServer.py

```diff
@@ -424,9 +424,9 @@ def _load(self, decoded_obj: dict, game_data_packages: typing.Dict[str, typing.A
                              for player, loc_data in decoded_obj["er_hint_data"].items()}
 
         # load start inventory:
-        for slot, item_codes in decoded_obj["precollected_items"].items():
-            self.start_inventory[slot] = [NetworkItem(item_code, -2, 0) for item_code in item_codes]
-
+        for slot, items in decoded_obj["precollected_items"].items():
+            self.start_inventory[slot] = [NetworkItem(item[0] if type(item) == tuple else item, -2, 0,
```

Comment:
```suggestion
	    if isinstance(items[0], tuple):
	        self.start_inventory[slot] = [NetworkItem(item_code, -2, 0, flags) for item_code, flags in items]
	    else:
	        self.start_inventory[slot] = [NetworkItem(item_code, -2, 0) for item_code in items]
```
maybe?

Reply:
It mostly would work but if the list is empty it will fail.